src/components/plots/styling/SelectData.jsx

```diff
@@ -134,12 +133,14 @@ SelectData.propTypes = {
   config: PropTypes.object,
   onUpdate: PropTypes.func.isRequired,
   cellSets: PropTypes.object.isRequired,
-  axisName: PropTypes.oneOf(['x', 'y']),
+  firstSelectionText: PropTypes.string,
+  secondSelectionText: PropTypes.string,
 };
 
 SelectData.defaultProps = {
   config: null,
-  axisName: 'y',
+  firstSelectionText: 'Select the cell sets or metadata that cells are grouped by',
+  secondSelectionText: 'Select the cell sets or metadata to be shown as data',
```

Comment:
do we really need to have that difference in texts? Can't we use the same `firstSelectionText` and `secondSelectionText` everywhere and get rid of these props?

Reply:
No. For example, in the violin plot, the 1st selection is used to show the x axis, but for the dot plot it is used for the y axis. It is contextually different for different plots